kong/plugins/aws-lambda/aws-serializer.lua

```diff
@@ -86,6 +87,16 @@ return function(ctx, config)
   local uri = var.upstream_uri or var.request_uri
   local path = uri:match("^([^%?]+)")  -- strip any query args
 
+  -- A resource identifier that is compatible with both the old/new router.
+  local resource
+  local route = kong.router.get_route()
+  if route and route.expression then
+    resource = route.expression
```

Comment:
Wouldn't UUID be more appropriate here? I don't know if `expression` is helpful to the user in this case.

Reply:
According to [the AWS proxy integration document](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format), the `resource` field is more like a path value than an ID. And there are also other fields inside the `requestContext` struct that reflect the resource basic info, for example, `resourceId` and `resourcePath`. I think it would be better if the `resource` field equals to `resourcePath`, and let `resourceId` be the route entity's UUID

Reply:
But anyway, there is no official definition of the `resource` field, and we're merely mimic-ing the AWS product's behaviour, so I'm just trying to imitating with the most alike way here 😄